app/services/scorecard_result_service.js

```diff
@@ -1,9 +1,9 @@
 const scorecardResultService = (() => {
   return {
-    isAllowToFinish,
+    isAllowToSaveDraft
   };
 
-  function isAllowToFinish(scorecard, criterias) {
+  function isAllowToSaveDraft(scorecard, criterias) {
```

Comment:
@limkimsan please consider to refactor to isSaveAble(scorecard) { ... } as we used stateful so every update criterias(input) it writes to scorecard -> so isSaveAble should needs only a scorecard object which is currently working on -> from that scorecard we should get criterias, ..etc. as we wish